uwb_platform_interface/lib/uwb_platform_interface.dart

```diff
@@ -0,0 +1,41 @@
+import 'package:plugin_platform_interface/plugin_platform_interface.dart';
+
+///Contains all the calls that needs to be implemented in the platform specific plugins
+abstract class UwbPlatform extends PlatformInterface {
+  /// Constructs a UwbPlatform.
+  UwbPlatform() : super(token: _token);
+
+  static final Object _token = Object();
+
+  /// The instance of [UwbPlatform] to use.
+  /// Must be set before accessing.
```

Comment:
According to the [documentation style guide](https://dart.dev/guides/language/effective-dart/documentation#do-separate-the-first-sentence-of-a-doc-comment-into-its-own-paragraph), it is best practice to keep an empty comment line after the first sentence (which acts as the summary) of a doc comment:

```suggestion
  /// The instance of [UwbPlatform] to use.
  ///
  /// Must be set before accessing.
```